server/events/runtime/apply_step_runner.go

```diff
@@ -38,7 +37,9 @@ func (a *ApplyStepRunner) Run(ctx models.ProjectCommandContext, extraArgs []stri
 
 	ctx.Log.Info("starting apply")
 	var out string
-	if a.isRemotePlan(contents) {
+
+	// TODO: Leverage PlanTypeStepRunnerDelegate here
+	if IsRemotePlan(contents) {
```

Comment:
I the plan to do this later? Should we track it in jira?

Reply:
Yeah this is more cleanup work which I don't want to do now.  Probably ok to not track refactors in JIRA unless they are essential for a feature.